src/static/dataloader.py

```diff
@@ -1,56 +1,112 @@
 import json
 import os
-
 import torch
 import xlrd
 
 PROJECT_PATH = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 
 
 class DataLoader:
-    def __init__(self):
+    def __init__(self, model: str):
+        self.model = model.lower()  # Standardize country string
+
+        # Dictionary mapping for the models and corresponding data files
+        model_data_files = {
```

Comment:
Create a JSON file and put this dictionary there. Load the JSON file and this structure will be available easily.